### ISSUE TITLE
1.16 model data

### ISSUE THREAD

**Issue by TheCrunchy** [Hazebyte/CrateReloaded#470]
![image](https://user-images.githubusercontent.com/13859025/86543159-f0980580-bf13-11ea-9d98-e10fa81b0aec.png)

Model data doesnt work in 1.16
Example that works in 1.15
- 'unique:(),     chance:(200),          item:(slime_ball{CustomModelData:1} 3 name:&6Bronze_Relic lore:&3Tier_1 durability:10), display:(slime_ball{CustomModelData:1} 3 name:&6Bronze_Relic lore:&3Tier_1 durability:10)'

**Comment by mibby** [comment]
@imWillX I'm also able to confirm that CustomModelData does not seem to be working correctly with the latest CrateReloaded on 1.16.

CrateReloaded Pro 2.0.29
Paper dev 101 (Spigot 1.16 Fork)

Example Item
```
    - chance:(1.0), display:(DIAMOND_SHOVEL{CustomModelData:398} 1 name:&aAccessories_Pack! lore:&f{chance} unbreakable:true hide:true), permission:(hat.accessories), cmd:(/lp user {player} permission set hat.accessories), broadcast:({prefix} {player} has opened a hat crate and unlocked accessories pack!)
```

Spawning in an item via ``/i DIAMOND_SHOVEL 1 {CustomModelData:398}`` _does_ correctly show the texture in-game, indicating it is a CrateReloaded problem and not a resource pack one.

**Comment by imWillX** [comment]
Fix `2.0.30`.
